tests/unit/utils.py

```diff
@@ -199,6 +199,9 @@ def assert_from_dict(obj, dct, cls=None):
         obj_from_dict.created_at = 1
         obj.created_at = 1
 
+    if obj_from_dict != obj:
```

Comment:
nit: This could arguably be put in the assert statement:
```python
assert obj_from_dict == obj, f"--- obj_from_dict: {obj_from_dict}\n--- obj: {obj}"
```

Reply:
Removed.